README.md

```diff
@@ -40,6 +40,10 @@ Jaeger is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CN
   * [Kubernetes templates](https://github.com/jaegertracing/jaeger-kubernetes)
   * [OpenShift templates](https://github.com/jaegertracing/jaeger-openshift)
 
+The Jaeger contributors meet weekly, and everyone is welcome to join.
```

Comment:
I think we agreed every two weeks.

Reply:
Indeed. I'll change this to WIP until we have a calendar link. Saying "every two weeks starting on Nov 17th" will make people crazy in, say, March :)